src/randomenv.cpp

```diff
@@ -59,7 +59,9 @@
 #include <sys/auxv.h>
 #endif
 
+#ifndef _MSC_VER
```

Comment:
?

Reply:
The `extern char** environ;` declaration is not necessary when compiling with MSVC, and it causes warning [C4273](https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-1-c4273?view=msvc-170) "inconsistent dll linkage".

Reply:
Maybe fix / workaround the warning, rather than adding more #ifdefs to our RNG code?

Reply:
> Maybe fix ... the warning

According to the [docs](https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-1-c4273?view=msvc-170):

> To fix it, delete the redeclaration